src/thermo/BinarySolutionTabulatedThermo.cpp

```diff
@@ -90,26 +91,50 @@ void BinarySolutionTabulatedThermo::initThermo()
         size_t N = x.size();
         vector_fp h = table.convertVector("enthalpy", "J/kmol", N);
         vector_fp s = table.convertVector("entropy", "J/kmol/K", N);
+        vector_fp vbar;
+        // Check for partialMolarVolume tag in tabulatedThermo table, take m_speciesMolarVolume[0] if false (for backward compatibility)
+        if (table.hasKey("partialMolarVolume")){
+        	vbar = table.convertVector("partialMolarVolume", "m3/kmol", N);
+        	for(size_t i=0; i<N; i++){
+        		vbar[i]+=m_speciesMolarVolume[1];
+        	}
+        }
+        else{
+        	vbar.resize(x.size());
+        	for(size_t i=0; i<N; i++){
+        		vbar[i]=m_speciesMolarVolume[0];
```

Comment:
Please only use spaces for indentation, with 4 spaces per indent.

Reply:
@dschmider-HSOG Depending on what code editor you are using, you can likely set this as a preference, so that a tab automatically inserts 4 spaces.